src/test/java/io/github/bobocodebreskul/context/scan/RecursiveClassPathAnnotatedBeanScannerTest.java

```diff
@@ -101,9 +102,9 @@ void given_EmptyPackage_When_Scan_Then_RegisterNothing() {
   @Order(4)
   void given_EmptyInput_When_Scan_Then_RegisterDoNothing() {
     // when
+    doNothing().when(scanUtils).validatePackagesToScan();
     annotatedBeanScanner.scan();
     // verify
-    then(scanUtils).shouldHaveNoInteractions();
```

Comment:
Instead of removing it better to validate the call.

Reply:
As for me this test is superfluous, since when an empty value is passed, an exception will be called

Reply:
turned back